app/controllers/safety_sketches_controller.rb

```diff
@@ -0,0 +1,40 @@
+class SafetySketchesController < ApplicationController
```

Comment:
@aerugo Hm can you take me through the differences between SafetySketches and Images? It seems like there's a whole lot of duplication between the two models, and I wonder if we could get away with simply adding a new way to relate a camp to an image as a SafetySketch

Reply:
They are now, but down the line they won't be. SafetySketches will probably need to take PDFs and technical drawing documents as filetypes, and might be annotated. Another option is that safety sketches might not be possible to delete after uploading once approved. It's also possible that we will process SafetySketch photos differently, and perhaps don't compress them as much. I don't have all the requirements yet, so that's why I'm just duplicating the code for now.